Without this patch, pic30-elf-gcc complains:
pic30-elf-gcc: installation problem, cannot exec `pic30-elf-cc1': No such file or directory

--- pic30-gcc/gcc-4.0.2/gcc-4.0.2/gcc/config/pic30/pic30.h.orig	2011-11-27 07:25:21.000000000 +0000
+++ pic30-gcc/gcc-4.0.2/gcc-4.0.2/gcc/config/pic30/pic30.h	2011-11-27 07:27:11.000000000 +0000
@@ -287,15 +287,6 @@
 #undef   STARTFILE_SPEC
 #define   STARTFILE_SPEC   ""
 
-/* making STANDARD_EXEC_PREFIX and STANDARD_BINDIR_PREFIX point to the same
-   directory will cause make_relative_paths to make no change - ie look in the 
-   gcc executable's directory.  */
-#undef STANDARD_EXEC_PREFIX
-#undef STANDARD_BINDIR_PREFIX
-#define STANDARD_EXEC_PREFIX "/bin/"
-#define STANDARD_LIBEXEC_PREFIX "/bin/"
-#define STANDARD_BINDIR_PREFIX "/bin/bin"
-
 /* By default, the GCC_EXEC_PREFIX_ENV prefix is "GCC_EXEC_PREFIX", however
    in a cross compiler, another environment variable might want to be used
    to avoid conflicts with the host any host GCC_EXEC_PREFIX */
